.github/workflows/build-frontend-development.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow:
+# - Sets up Node.js with a yarn cache
+# - Installs packages via yarn, using yarn.lock
+# - Builds the frontend bundle
+# - Commits the frontend bundle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Frontend CI/CD [dev]
+
+on:
+  push:
+    branches: [ "development" ]
+    paths: 'frontend/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

Comment:
No real reason to use a matrix here. We're only ever going to want to run this with one version. So you can just hardcode `node-version: '18'` below, and get rid of the whole `strategy` section.